tensorflow/contrib/tensorrt/resources/trt_allocator_test.cc

```diff
@@ -48,11 +48,11 @@ TEST(TRTAllocatorTest, Align) {
         513ul, 700ul, 12345ul, 1ul << 32}) {
     for (uint64_t alignment = 1; alignment <= space * 4; alignment *= 2) {
       for (const uintptr_t ptr_val :
-           {1ul, alignment == 1 ? 1ul : alignment - 1, alignment, alignment + 1,
-            alignment + (alignment / 2)}) {
+           {1ull, alignment == 1 ? 1ull : alignment - 1, alignment,
```

Comment:
This change was needed due to the ternary boolean operator returning `unsigned long` vs `unsigned long long`. Forcing the `1ul` to `unsigned long long` seemed to resolve the issue.

I see the `nomac` tag in the build files. However, am I supposed to explicitly add the `nomac` tag when running my tests? I had run `bazel test -c opt -- //tensorflow/... -//tensorflow/compiler/... -//tensorflow/contrib/lite/...`.

Reply:
Yeah `bazel test` with broad wildcards isn't reliable since we don't run presubmits to verify that it works. But fixing the build here SGTM since it's relatively simple.

Reply:
@jaingaurav have you tested these changes internally? I patched them and encountered build errors.

Reply:
@aaroey: I hadn't tested it on linux. But I will do so. However given your comment about the `nomac` tag this change can probably be reverted.